inc/export.class.php

```diff
@@ -503,6 +503,24 @@ static function getAllUsedItemsForUser($ID) {
          }
       }
 
+      // Consumables
+      $consumables = "SELECT `glpi_consumableitems`.`name`,
```

Comment:
better use iterator instead of raw queries ;)

Reply:
done